pkgs/by-name/gv/gvmd/package.nix

```diff
@@ -0,0 +1,114 @@
+, libtasn1
+, postgresql
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gvmd";
+  version = "23.4.0";
+
+  src = fetchFromGitHub {
+    owner = "greenbone";
+    repo = "gvmd";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-Bxq6o98viMMwt2eoiZgPDTYqD/z6jRWYLgH/ZJvkpV4=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    gvm-libs
+    graphviz
+    mount
+    gnutls
+    glib
+    git
+    doxygen
+    pcre2
+    p11-kit
+    libbsd
+    libidn2
+    libtasn1
+    icu
+    libical
+    xml-tooling-c
+    gpgme
+    libxslt
+    postgresql
+  ];
+
+  buildPhase = ''
+    substituteInPlace $src/src/sql_pg.c --replace "#include postgresql/libpq-fe.h" "#include ${postgresql}/include/libpq-fe.h"
+    mkdir -p $out/{bin,etc,sbin,lib,usr/{share,include},run/gvmd/gvmd.pid}
+    mkdir -p \
+      $out/var/log/gvm \
+      $out/var/lib/openvas/plugins \
+      $out/var/lib/gvm/cert-data \
+      $out/var/lib/gvm/users \
+      $out/var/lib/gvm/scap-data \
+      $out/var/lib/gvm/CA \
+      $out/var/lib/gvm/private/CA \
+      $out/var/lib/gvm/data-objects \
+      $out/var/lib/gvm/feed-update.lock
+    cmake -DSYSCONFDIR=$out/etc \
+          -DLOCALSTATEDIR=$out/var \
+          -DINCLUDEDIR=$out/usr/include \
+          -DDATADIR=$out/usr/share \
+          -DGVM_SYSCONF_DIR=$out/etc/gvm \
+          -DGVM_DATA_DIR=$out/usr/share/gvm \
+          -DGVMD_DATA_DIR=$out/usr/share/gvmd \
+          -DGVM_STATE_DIR=$out/var/lib/gvm \
+          -DGVMD_STATE_DIR=$out/var/lib/gvm/gvmd \
+          -DGVM_LOG_DIR=$out/var/log/gvm \
+          -DGVM_SCAP_RES_DIR=$out/usr/share/gvm/scap \
+          -DGVM_CERT_RES_DIR=$out/usr/share/gvm/cert \
+          -DGVM_CA_DIR=$out/var/lib/gvm/gvmd/trusted_certs \
+          -DGVM_LIB_INSTALL_DIR="''${LIBDIR}" \
+          -DGVM_SCANNER_CERTIFICATE=$out/var/lib/CA/servercert.pem \
+          -DGVM_SCANNER_KEY=$out/var/lib/private/CA/serverkey.pem \
+          -DGVM_CLIENT_CERTIFICATE=$out/var/lib/CA/clientcert.pem \
+          -DGVM_CLIENT_KEY=$out/var/lib/private/CA/clientkey.pem \
+          -DGVM_CA_CERTIFICATE=$out/var/lib/CA/cacert.pem \
+          -DGVMD_RUN_DIR=$out/run/gvmd \
+          -DGVMD_PID_PATH=$out/run/gvmd/gvmd.pid \
+          -DGVM_FEED_LOCK_PATH=$out/var/lib/gvm/feed-update.lock \
+          -DGVM_NVT_DIR=$out/var/lib/openvas/plugins/ \
+          -DDATA_OBJECTS_FEED_DIR=$out/var/lib/gvm/data-objects \
+          -DGVMD_FEED_DIR=$out/var/lib/gvm/data-objects/gvmd/''${GMP_VERSION_FEED} \
+          -DGVM_CERT_DATA_DIR=$out/var/lib/gvm/cert-data \
+          -DSYSTEMD_SERVICE_DIR=$out/lib/systemd/system \
+          -DGVM_SCAP_DATA_DIR=$out/var/lib/gvm/scap-data .
+    mkdir -p $out/etc/systemd/system/multi-user.target.wants
+    ln -s $out/lib/systemd/system/gvmd.service $out/etc/systemd/system/multi-user.target.wants/gvmd.service
+  '';
+
+  meta = with lib; {
+    description = "Greenbone Vulnerability Manager - The database backend for the Greenbone Community Edition";
```

Comment:
```suggestion
    description = "The central management service between security scanners and the user clients";
```

Please don't repeat the package name. Also repository description is not always good.